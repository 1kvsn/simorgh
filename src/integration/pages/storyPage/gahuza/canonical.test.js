/**
 * @service gahuza
 * @pathname /gahuza/23313911
 * @runScripts false
 */

import runCanonicalIncludeTests from '../canonicalIncludeTests';
import runCanonicalTests from '../canonicalTests';

describe('Canonical', () => {
  describe(pageType, () => {
    runCanonicalIncludeTests();
    runCanonicalTests(service);
  });
});
