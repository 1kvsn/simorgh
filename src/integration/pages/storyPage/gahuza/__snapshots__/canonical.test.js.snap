// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical Story Page Includes I can see a fallback image when there are disallowed scripts 1`] = `
<img
  alt="Map: St. Albans"
  class="css-13lcmnl-StyledImg e1enwo3v0"
  id="include-1"
  src="https://www.test.bbc.com/ws/includes/idt2/3686672a-0a64-4cdb-91c6-ae9ed0f214f7/image/816"
  srcset="https://www.test.bbc.com/ws/includes/idt2/3686672a-0a64-4cdb-91c6-ae9ed0f214f7/image/640 640w,https://www.test.bbc.com/ws/includes/idt2/3686672a-0a64-4cdb-91c6-ae9ed0f214f7/image/816 816w"
/>
`;

exports[`Canonical Story Page Includes IDT2 I can see a "dataPic" 1`] = `
<script
  src="https://news.test.files.bbci.co.uk/include/idt2/static/js/dataPic.8f264bc5.js"
  type="text/javascript"
/>
`;
